pkg/sriov/sriov.go

```diff
@@ -284,6 +287,19 @@ func (s *sriovManager) FillOriginalVfInfo(conf *sriovtypes.NetConf) error {
 		return fmt.Errorf("failed to find vf %d", conf.VFID)
 	}
 	conf.OrigVfState.FillFromVfInfo(vfState)
+
+	if len(pfLink.Attrs().Vfs) < conf.VFID {
```

Comment:
You don't need this part. This function is about the attributes of VFs obtained via the PF. Besides, conf.OrigVfState.EffectiveMAC is already obtained in SetupVF (line 85).

The protection you have above is not needed either. vfState (above) will be nil when VFID is not found. An error is returned for this case.

Reply:
done